### ISSUE TITLE
Ability to dock tabs side by side

### ISSUE THREAD

**Issue by malsamaxyz** [TablePlus/TablePlus#730]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):

N/a

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):

143

3. The steps to reproduce this issue:

Especially when creating a query it would be handy to see the contents and/or structure of a table. Now I have to keep switching between tabs. Docking the tabs horizontally/vertically would be great.

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
TablePlus will suggest all table columns if you type: TablePlus + dot. 
Ex:
1. SELECT * FROM comments where comments. <- the columns will be listed
2. SELECT * FROM comments as c where c. <- the same.

I don't know there is any difference between a column list on the sidebar and the column list inside the auto-completion popup? The column list on the auto-completion popup has fuzzy matching so you can easily filter it without remember the name.

Did I miss something?

**Comment by malsamaxyz** [comment]
> Did I miss something?

Yes, I know I can use auto-complete, but there are scenarios that it's handy to see a overview of the data and/or structure (incl. types, default value Y/N) while constructing a query.

**Comment by Sulemanerator** [comment]
+1 for this -- when creating new reports in unfamiliar databases, it is useful to have the ability to dock tabs, especially when creating complex joins
